service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/ApplicationDocumentCheckerTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.enums.ApplicationDocumentType;
+import uk.gov.hmcts.reform.fpl.model.ApplicationDocument;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@ExtendWith(MockitoExtension.class)
+class ApplicationDocumentCheckerTest {
+
+    @InjectMocks
```

Comment:
There are no mocks to inject so this annotation and the class level one can be removed

Reply:
doesn't seem to work when removing - will keep as is, i copied this from other checker tests.